### DISCUSSION TITLE
When will we have the next release?

### DISCUSSION THREAD

**Discussion by kushaldas** [celery/kombu#1662] [General]
Asking for the next release as we can not successfully rebuild the `5.2.4` release, the cause is https://github.com/celery/kombu/pull/1652#issuecomment-1442300039

@auvipy any tips?

**Comment by auvipy** [comment]
is it possible for you to downgrade the setuptools version for 5.2.x series? I was suppose to release v5.3.0b3 today but need some prior work on CI before that so I think couple of days for a new release from man branch.
